src/scripts/browserDeviceProfile.js

```diff
@@ -345,6 +345,23 @@ import browser from './browser';
         return 2;
     }
 
+/**
+ * Checks if the web engine supports secondary audio.
+ * @param {HTMLVideoElement} videoTestElement The video test element
+ * @returns {boolean} _true_ if the web engine supports secondary audio.
+ */
+export function canSecondaryAudio(videoTestElement) {
```

Comment:
Maybe this could be renamed to be similar to the other functions?

```suggestion
export function supportsSecondaryAudio(videoTestElement) {
```

Reply:
We have variable with the same name in the `browserDeviceProfile` - I was afraid of the shadowing error.

Reply:
`canSupportSecondaryAudio`? :sweat_smile:

Reply:
Renamed to `canPlaySecondaryAudio`